# What: Fix time check typo. 
#       This patch is based on the following PR: https://github.com/openbabel/openbabel/pull/2493
# Error message:
#       error: clock was not declared in this scope
#       65 |       start= clock();
#          |              ^~~~~
# Author: maxim-masterov (SURF)
diff -Nru openbabel-openbabel-3-1-1.orig/src/config.h.cmake openbabel-openbabel-3-1-1/src/config.h.cmake
--- openbabel-openbabel-3-1-1.orig/src/config.h.cmake	2023-10-09 17:43:35.852332000 +0200
+++ openbabel-openbabel-3-1-1/src/config.h.cmake	2023-10-09 17:44:32.175471000 +0200
@@ -182,8 +182,8 @@
 #define OB_MODULE_PATH "@OB_MODULE_PATH@"
 
 #ifndef TIME_WITH_SYS_TIME
-  #ifdef HAVE_SYS_TIME
-    #ifdef HAVE_TIME
+  #ifdef HAVE_SYS_TIME_H
+    #ifdef HAVE_TIME_H
       #define TIME_WITH_SYS_TIME 1
     #else
       #define TIME_WITH_SYS_TIME 0
